go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/opencontainers/image-spec
 
-go 1.18
+go 1.19
```

Comment:
Given that this is now officially defined as the lowest version _supported_ by the module (not the recommended/preferred version, as it's been previously), should we try to be more conservative than what Go officially supports in case we've got some lagging consumers?

We don't have a _lot_ of Go code in here, and I don't think there have been any really compelling Go features/changes for the functionality we actually use, so I guess this is functionally going to be dictated by our few dependencies more than it is by our own code. :thinking:

Reply:
That's fair. I also noticed that our GHA has 3 versions being tested (1.18, 1.19, and 1.20) so this update can wait until 1.21 comes out.